src/explore-education-statistics-common/src/components/Modal.module.scss

```diff
@@ -46,3 +46,13 @@
     outline-offset: -$govuk-focus-width;
   }
 }
+
+.content {
+  margin-bottom: govuk-spacing(5);
+  max-height: 50vh;
```

Comment:
Would it be possible to bump this up some more? Something like 70vh could be better.

Reply:
This has been removed now.